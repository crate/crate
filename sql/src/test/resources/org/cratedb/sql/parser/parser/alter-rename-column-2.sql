ALTER TABLE t RENAME COLUMN c AS d