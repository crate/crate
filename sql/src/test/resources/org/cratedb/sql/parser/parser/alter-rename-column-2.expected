org.cratedb.sql.parser.parser.AlterTableNode@71172d81
name: t
statementType: ALTER TABLE
objectName: t
compressTable: false
sequential: false
truncateTable: false
purge: false
defragment: false
truncateEndOfTable: false
updateStatistics: false
updateStatisticsAll: false
indexNameForUpdateStatistics: null
tableElementList: 	
	org.cratedb.sql.parser.parser.TableElementList@28acaa28

	[0]:	
	org.cratedb.sql.parser.parser.AlterTableRenameColumnNode@5fd73cf4
	name: c
	elementType: AT_RENAME_COLUMN